packages/web/src/scss/components/Modal/index.html

```diff
@@ -719,16 +719,224 @@ <h2 id="example-custom-height-title" class="ModalHeader__title">Modal with Custo
+              const breakpointSuffix = breakpoint === 'mobile' ? '' : `-${breakpoint}`;
+              const modalElement = document.querySelector(selector);
+              const modalHeightValueElement = document.querySelector(`#custom-max-height-${breakpoint}-value`);
+
+              modalElement.style.setProperty(`--modal-max-height${breakpointSuffix}`, `${event.target.value}px`);
+              modalHeightValueElement.textContent = `${event.target.value} px`;
+            };
+          </script>
+          <form>
+            <fieldset class="Stack Stack--hasSpacing mb-800" style="border: 0;">
+              <legend hidden>Mobile</legend>
+              <label for="custom-height-mobile-enabled" class="Checkbox">
+                <input
+                  type="checkbox"
+                  id="custom-height-mobile-enabled"
+                  class="Checkbox__input"
+                  onchange="toggleCustomHeight(event, '#example-custom-height > .ModalDialog', 'mobile')"
+                  checked
+                  autocomplete="off"
+                />
+                <span class="Checkbox__text">
+                  <span class="Checkbox__label">Mobile</span>
+                </span>
+              </label>
+              <div class="Grid" style="column-gap: var(--spirit-space-600)">
+                <label for="custom-height-mobile" class="GridItem" style="--grid-item-column-start: 1; --grid-item-column-end: 6; --grid-item-column-end-tablet: 4;">
+                  Height
+                </label>
+                <span id="custom-height-mobile-value" class="GridItem" style="--grid-item-column-start: 6; --grid-item-column-end: 13; --grid-item-column-start-tablet: 4; --grid-item-column-end-tablet: 7;">
+                  400 px
+                </span>
+                <input
+                  type="range"
+                  id="custom-height-mobile"
+                  class="GridItem"
+                  style="--grid-item-column-start: 1; --grid-item-column-end: 13; --grid-item-column-start-tablet: 7;"
+                  onchange="setModalHeight(event, '#example-custom-height > .ModalDialog', 'mobile')"
+                  min="200"
+                  max="1000"
+                  step="100"
+                  value="400"
+                  autocomplete="off"
+                />
+              </div>
+              <div class="Grid" style="column-gap: var(--spirit-space-600)">
+                <label for="custom-max-height-mobile" class="GridItem" style="--grid-item-column-start: 1; --grid-item-column-end: 6; --grid-item-column-end-tablet: 4;">
+                  Max height
+                </label>
+                <span id="custom-max-height-mobile-value" class="GridItem" style="--grid-item-column-start: 6; --grid-item-column-end: 13; --grid-item-column-start-tablet: 4; --grid-item-column-end-tablet: 7;">
+                  600 px
+                </span>
+                <input
```

Comment:
This probably won't be <GridItem elementType="input" in twig :D

Reply:
`Unsafe_Slider` ?